### PR TITLE
Implement support for Aqua Computer Farbwerk

### PR DESCRIPTION
**Pull request by IreNox**

### CODE REVIEW

File: LibreHardwareMonitorLib/Hardware/Controller/AquaComputer/Farbwerk.cs [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+// If a copy of the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
+// Copyright (C) LibreHardwareMonitor and Contributors.
+// All Rights Reserved.
+
+using System;
+using HidSharp;
+
+namespace LibreHardwareMonitor.Hardware.Controller.AquaComputer;
+//TODO:
+//Implement set RGB Controls
```
Comment (PhyxionNL) [inline LibreHardwareMonitorLib/Hardware/Controller/AquaComputer/Farbwerk.cs:11]: Do yous till want to do this?

Comment (IreNox) [reply LibreHardwareMonitorLib/Hardware/Controller/AquaComputer/Farbwerk.cs:11]: No. I've no RGB hardware. So I can't test it. But maybe someone else wants to add that later.

Comment (PhyxionNL) [reply LibreHardwareMonitorLib/Hardware/Controller/AquaComputer/Farbwerk.cs:11]: OK,  please format this document to follow code style and I'll merge :)

Comment (PhyxionNL) [reply LibreHardwareMonitorLib/Hardware/Controller/AquaComputer/Farbwerk.cs:11]: Thanks!

**Pull request merged**